scripts/prepare-preview-builds.sh

```diff
@@ -33,7 +33,7 @@ echo "Preparing manifests..."
 while IFS=$'\t' read -r location name; do
   echo "- $name"
   prepare-preview-manifest "$location/package.json"
-done < <(yarn workspaces list --no-private --json | jq --slurp --raw-output 'map(select(.location != ".")) | map([.location, .name]) | map(@tsv) | .[]')
+done < <(yarn workspaces list --json | jq --slurp --raw-output 'map(select(.location != ".")) | map([.location, .name]) | map(@tsv) | .[]')
```

Comment:
Removing `--no-private` flag to process all workspaces, including private ones in the `apps/` directory. This ensures all workspace dependencies are updated consistently, preventing "[workspace not found](https://github.com/MetaMask/metamask-design-system/pull/79/files#r1831802757)" errors that occur when private packages reference workspace dependencies that weren't processed.